### PR TITLE
Added documentation for Review and image dataset creadtion/predictions

### PR DESCRIPTION
**Pull request by Scott771**
Added examples for how to create an image dataset from local images and how to get predictions for an image model. 

Removed deprecated ListWorkflowsForDataset from documentation.

### CODE REVIEW

File: examples/create-image-dataset.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+]
+image_dataset.to_csv("./image_dataset.csv", index=False)
+
+# Use the CSV you created (like above) to create the dataset
+dataset = client.call(
+    CreateDataset(
+        name="My Image Dataset",
+        files="./image_dataset.csv",
+        from_local_images=True,
+        image_filename_col="image_files",  # specify the columns containing the images
```
Comment (jkgenser) [inline examples/create-image-dataset.py:35]: only one column can contain an image

Comment (Scott771) [reply examples/create-image-dataset.py:35]: just a spelling mistake in the comment, will fix

File: docs/_sources/review-intro.rst.txt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    
+    from indico.queries import WorkflowSubmission
+
+    submissions = client.call(
+        WorkflowSubmission(files=["./path/to/sample.pdf"], workflow_id=52, submission=True)
+    ) 
+    print(submissions[0].id)
+
+WorkflowSubmissions will return a list of Submission objects, each with a unique identifier and job id. 
+In the background, the documents are OCR'd (when applicable) and been passed to workflow's models. Once
```
Comment (jkgenser) [inline docs/_sources/review-intro.rst.txt:35]: and [have] been

File: docs/_sources/review-intro.rst.txt [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    submissions = client.call(ListSubmissions(workflow_ids=[52], filters=SubmissionFilter(status="COMPLETE")))
+    # Generate the result for the first completed submission
+    job = client.call(GenerateSubmissionResult(submissions[0].id))
+    result_url = client.call(JobStatus(id=job.id, wait=True))
+    result = client.call(RetrieveStorageObject(result_url.result))
+    # Finally, mark the submission as retrieved
+    client.call(UpdateSubmission(submissions[0].id, retrieved=True))
+
+
+This is what a sample of what the "result" object in the snippet above might look like::
```
Comment (jkgenser) [inline docs/_sources/review-intro.rst.txt:56]: You can remove "hat" in "This is what a sample of what"

File: examples/create-image-dataset.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+"""
+Creating an Image Dataset from Local PNGs/JPGs
+
+There are two ways two create a data of PNGs/JPGs depending on whether the images are
```
Comment (jkgenser) [inline examples/create-image-dataset.py:4]: In the near future there will be an additional way which will be to upload the image files directly as is possible to upload pdfs. This means we don't need to create a temporary csv using the client

**Review by jkgenser — APPROVED** [review]

**Pull request merged**